packages/playground/src/api/documents/types.gen.ts

```diff
@@ -1,3 +1,7 @@
+// Do not modify this file!
```

Comment:
Can you change this to be same as https://github.com/cognitedata/cognite-sdk-js/blob/4394e60273fa4fe33be731289beb2b09dac57f31/packages/stable/src/api/documents/types.gen.ts#L1-L4 ?

Reply:
Fixed